timemachine/fe/rbfe.py

```diff
@@ -416,11 +413,9 @@ def estimate_relative_free_energy(
         If None, return only the end-state frames. Otherwise if not None, use only for debugging, and this
         will return the frames corresponding to the idxs of interest.
 
-    n_eq_steps: int
-        Number of equilibration steps for each window.
-
-    steps_per_frame: int
-        The number of steps to take before collecting a frame
+    md_params: MDParams
+        Parameters to determine the equilibration and production MD. Defaults to 400 global steps per frame, 1000 frames and 10k
```

Comment:
nit: to determine -> for

Reply:
Fixed in [6f2ae30](https://github.com/proteneer/timemachine/pull/1026/commits/6f2ae30b84861f20fee58791fa241e945776728c)